app/models/user.rb

```diff
@@ -249,6 +249,10 @@ def full_name
     "#{self.first_name} #{self.last_name}"
   end
 
+  def username
```

Comment:
As you can see, `username` is a virtual attribute built upon the user's first and last name. So when a user updates her full name she is updating her username accordingly. _Participa_ does not make use of this attribute in terms of authentication/authorization, so this scenario would not have further consequences. But bear in mind that oauth2 clients like _Discourse_ use it as a public identifier for the user among the community. It would be undesirable that an update to a user's fullname could impersonate another user in Discourse.